doc/schemas/lightning-listaddresses.json

```diff
@@ -0,0 +1,162 @@
+{
+  "$schema": "../rpc-schema-draft.json",
+  "type": "object",
+  "added": "v24.11",
+  "rpc": "listaddresses",
+  "title": "Command to list all addresses issued by the node to date",
+  "description": [
+    "The **listaddresses** RPC command provides a detailed list of all Bitcoin addresses that have been generated and issued by the Core Lightning node up to the current date."
+  ],
+  "categories": [
+    "readonly"
+  ],
+  "request": {
+    "required": [],
+    "additionalProperties": false,
+    "properties": {
+      "start": {
+        "type": "u64",
+        "description": [
+          "Starting index for listing addresses or searching for a particular address."
+        ],
+        "default": 0
+      },
+      "end": {
+        "type": "u64",
+        "description": [
+          "Ending index for listing addresses or searching for a particular address."
+        ],
+        "default": "Total number of addresses issued"
+      },
+      "address": {
+        "type": "string",
+        "description": [
+          "Any Bitcoin accepted type, including bech32."
```

Comment:
A more verbose description detailing that this address is being looked up would be nice

Reply:
Makes sense, updated!